### PR TITLE
Update post for privacy week 2023

### PR DESCRIPTION
**Pull request by davidfischer**
This is dated to go out on Monday.

- Link to our search engine teardown
- Mention DDG's mobile browser with tracker blocking
- Mention Brave search and browser
- Re-use some screenshots from our search engine teardown

After launching, create a redirect from the old post (`https://www.ethicalads.io/blog/2022/01/data-privacy-day-2022-6-privacy-first-alternatives/`) to this one.

## Screenshot

Screenshot 2023-01-20 at 15-57-50 Data Privacy Week 2023 - Privacy-first Alternatives

### CODE REVIEW

File: content/posts/2023-data-privacy-week.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
 ## Ad Blockers
 
-I'd used [AdBlockPlus](https://adblockplus.org/) for years without much thought,
-before I joined EthicalAds and started considering the industry as a whole.
-It's a simple, effective service which allows you control over not only which websites are allowed to show ads,
+About as important as the browser itself, are the additional privacy add-ons
+that keep you safe from ads and trackers.
+Many people like myself used [AdBlockPlus](https://adblockplus.org/) for years without much thought.
+It's a simple, effective service which gives you control over not only which websites are allowed to show ads,
 but it even allows you to greenlist certain YouTube and Twitch channels to run their ads
```
Comment (ericholscher) [inline content/posts/2023-data-privacy-week.md:42]:
```suggestion
but it even allows you to allow list certain YouTube and Twitch channels to run their ads
```

I think we're using this generally?

File: content/posts/2023-data-privacy-week.md [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
 so you can support your favorite creators.
 The creators of AdBlockPlus have gone on to form the [Acceptable Ads Committee](https://acceptableads.com/about/)
 who's [Acceptable Ads Standard](https://acceptableads.com/standard/) we follow
-as we believe in their mission of creating a sustainable and fair compromise between user preference and creator monetization.
+as EthicalAds believes in their mission of creating a sustainable and fair compromise between user preference and creator monetization.
```
Comment (ericholscher) [inline content/posts/2023-data-privacy-week.md:46]:
```suggestion
as EthicalAds believes in their mission of creating a sustainable between user preference and creator monetization.
```

Too many `and`'s.

**Review by ericholscher — APPROVED** [review]
Looks good 👍

File: content/posts/2023-data-privacy-week.md (hunk 2 above)
Comment (davidfischer) [reply content/posts/2023-data-privacy-week.md:46]: Sustainable compromise?

Comment (ericholscher) [reply content/posts/2023-data-privacy-week.md:46]: Ha yea, sorry.

**Pull request merged**